src/MeetingsAdapter.js

```diff
@@ -93,6 +93,18 @@ export default class MeetingsAdapter extends WebexAdapter {
     this.meetingControls = {};
   }
 
+  /**
+   * Returns a promise to send a response wether the
+   * local media had been added to the meeting successfully or not.x
+   *
+   * @param {string} meetingID  ID of the meeting to add the local media to.
+   * @returns {Promise.<Meeting>}
+   * @memberof MeetingsAdapter
+   */
+  addLocalMedia(meetingID) {
```

Comment:
```suggestion
  addLocalMedia(ID) {
```
For consistency with other methods